client/selfrepair/normandy_driver.js

```diff
@@ -86,6 +103,15 @@ export class HeartbeatEmitter {
 export default class NormandyDriver {
   constructor(uitour = Mozilla.UITour) {
     this._uitour = uitour;
+    this.setDurability();
```

Comment:
I think setting and checking this is the responsibility of the runtime, since we don't necessarily guarantee that only one driver is created per-run. If something causes the driver to be created multiple times, after the first time it will think that it is durable when in fact it is not.

I think the driver API we want is more along the lines of just having an `isStorageDurable()` method.